Casks/font/font-3/font-3270.rb

```diff
@@ -7,15 +7,19 @@
   homepage "https://github.com/rbanffy/3270font"
 
   livecheck do
-    url "https://github.com/rbanffy/3270font/releases/latest"
+    url :url
     regex(%r{v?(\d+(?:\.\d+)+)/3270[._-]fonts[._-](.*)\.zip}i)
```

Comment:
```suggestion
    regex(%r{v?(\d+(?:\.\d+)+)/3270[._-]fonts[._-](\h+)\.zip}i)
```

Some regex tweaks while we're working on this:

* The suffix appears to always be hex characters (e.g., `3270_fonts_fba25eb.zip`), so we should tighten this up to use `\h` instead of `.` (better representing our intention).
* Since `[._-]` isn't optional before the suffix, we should use `\h+` instead of `\h*` to make it non-optional as well. [If we wanted the suffix to be optional we would have to use `(?:[._-]\h+)?` instead and use `match[2] ? "#{match[1]},#{match[2]}" : match[1]` in the `strategy` block but that doesn't seem to be necessary here (upstream has used this filename format for all releases with an asset).]

Reply:
Got it, thanks!